drk-ontology.ttl

```diff
@@ -1,20 +1,339 @@
+    dcterms:source <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# The drk:allowsAssistanceDogs property describes if the place allows visitors to bring assistance/service dogs.
+# An assistance/service dog is a dog that receives specialized training to aid an individual with a disability in navigating everyday life (source: https://en.wikipedia.org/wiki/Assistance_dog).
+# Assistance/Service dogs are a specific category of service animals.
+drk:allowsAssistanceDogs
+    a                  rdf:Property, owl:DatatypeProperty ;
+    rdfs:subPropertyOf drk:allowsDogs, drk:allowsServiceAnimals ;
+    rdfs:domain        drk:AnimalPolicy ;
+    rdfs:range         schema:Boolean ;
+    rdfs:label         "Allows Assistance/Service Dogs"@en ;
+    rdfs:comment       "A property that describes if the place allows visitors to bring assistance/service dogs. Assistance dogs, also referred to as a service dogs, are a specific type of service animals that receive specialized training to aid an individual with a disability in navigating everyday life (source: https://en.wikipedia.org/wiki/Assistance_dog)."@en ;
+    dcterms:source     <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# The drk:allowsGuideDogs property describes if the place allows visitors to bring guide dogs.
+# Guide dogs are a specialized type of assistance dogs, trained to lead individuals who are blind or visually impaired around obstacles (source: https://en.wikipedia.org/wiki/Guide_dog).
+drk:allowsGuideDogs
+    a                  rdf:Property, owl:DatatypeProperty ;
+    rdfs:subPropertyOf drk:allowsAssistanceDogs ;
+    rdfs:domain        drk:AnimalPolicy ;
+    rdfs:range         schema:Boolean ;
+    rdfs:label         "Allows Guide Dogs"@en ;
+    rdfs:comment       "A property that describes if the place allows visitors to bring guide dogs. Guide dogs are a specialized type of assistance dogs, trained to lead individuals who are blind or visually impaired around obstacles (source: https://en.wikipedia.org/wiki/Guide_dog)."@en ;
+    dcterms:source     <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# The drk:WheelchairPlaces class describes the availability of reserved spots for wheelchairs at a place.
+drk:WheelchairPlaces
+    a              owl:Class ;
+    rdfs:label     "Wheelchair Places"@en ;
+    rdfs:comment   "A class that describes the presence of reserved spots for wheelchairs at a place."@en ;
+    dcterms:source <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# The drk:count property specifies the number of designated places for wheelchairs, for example, in a theater place or a cinema hall.
+drk:count
```

Comment:
I renamed the `drk:count` property to `drk:wheelchairPlacesCount` to address this.
It would be good to extend an existing generic "count" property. However, I couldn't find a suitable property to extend (yet).